src/app/shared/form-field/form-field.ts

```diff
@@ -17,7 +18,8 @@ export class TextFormField extends FormField {
        public label: string, 
        public defaultValue: string = '',
        public placeholder: string = '',
-       public validations: Validator[] = []) {
+       public validations: Validator[] = [],
+       public tooltip: Tooltip = new Tooltip()) {
```

Comment:
I would prefer to use the Type (we should always try that). Please check what other way is possible.